scenarios/centos-9/hci_ceph_backends.yml

```diff
@@ -5,6 +5,16 @@ cifmw_install_yamls_vars:
 
 cifmw_edpm_prepare_skip_crc_storage_creation: true
 
+pre_deploy:
+  - name: Disable openstack marketplace
+    type: playbook
+    source: disable_os_marketplace.yml
```

Comment:
We have removed this one since the last crc outage. Please update the PR removing this entry.

Reply:
Thanks for checking, it is removed now.